### DISCUSSION TITLE
umi4 如何使用最新 ahooks

### DISCUSSION THREAD

**Discussion by Mooo-star** [umijs/umi#12520] [Q&A]
umi4 如何使用最新 ahooks，项目使用了 umi 自带的 useRequest，但是想用 ahooks request 最新的一些方法，如何可以指定 ahooks 版本？

**Comment by fz6m** [comment]
不可以更改内置 useRequest 使用的库版本，但如果你想使用其他请求库，请额外安装额外配置。
